src/assets/TokenListController.test.ts

```diff
@@ -1,10 +1,12 @@
 import { stub } from 'sinon';
 import nock from 'nock';
+import contractmap from '@metamask/contract-metadata';
```

Comment:
[Q] I agree this is the best thing we can do for a fallback, but is there a more long term fallback option that doesn't require us to maintain this repo anymore? Perhaps if the user could supply their own token list.

Reply:
I ran that by @omnat  and I am quoting her reply here
`Yes that’s a valid point and in the long term that’s the plan to allow people to upload their own lists. But for now this fallback is better than no auto detection, even if we don’t keep updating this list`